### ISSUE TITLE
Validation for dynamic field names

### ISSUE THREAD

**Issue by malles** [pagekit/vue-form#3]
I find it impossible to init the validation on fields that have a dynamic name:

```
<input type="text" class="uk-form-width-large"
                   :attr="{name: fieldid, id: fieldid}"
                   v-model="dataObject.value"
                   v-validate:required="fieldRequired">
```

nor

```
<input type="text" class="uk-form-width-large"
                   :name="fieldid"
                   v-model="dataObject.value"
                   v-validate:required>
```

or other variants or orderings work.
The fieldname is not added to the form object passed back to the vm.

Could that be possible?

**Comment by janschoenherr** [comment]
should be resolved with: 54306f60556cdcf4ee75f83d1c0606ec109d6852

**Comment by malles** [comment]
Great, works with `:name="fieldid"`.

Note: does not work with `v-bind="{name: fieldid}"`
